external-crates/move/move-bytecode-verifier/src/meter.rs

```diff
@@ -9,6 +9,10 @@ use std::ops::Mul;
 /// Scope of meterinng
 #[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
 pub enum Scope {
+    // Metering is for transaction level
+    Transaction,
```

Comment:
My understanding is that the top-most scope is implicit in this API (i.e. the previous implementation did package level metering by grouping scopes up into an implicit top-level).  I personally don't have a problem with having an explicit `Transaction` level, but for it to match how things were organised before, would imply not having it.

Reply:
Yes this code is in preparation for a more explicit model.
with the current model, we do not need `Transaction` level scope.